openapi/openapi.yaml

```diff
@@ -58,4 +60,23 @@ paths:
     $ref: 'paths/lambdas/lambdas@collections@wearables.yaml'
   /lambdas/collections/wearables-by-owner/{address}:
     $ref: 'paths/lambdas/lambdas@collections@wearables-by-owner@{address}.yaml'
+  /lambdas/contentv2/scenes:
+    $ref: 'paths/lambdas/lambdas@contentv2@scenes.yaml'   
+  /lambdas/contentv2/parcel_info: 
+    $ref: 'paths/lambdas/lambdas@contentv2@parcel_info.yaml'
```

Comment:
Did you forget to add the route `/lambdas/contentv2/contents/:cid`?

Reply:
it was missing, just added the endpoint definition